README.md

```diff
@@ -4,195 +4,198 @@
+## Testing che-machine-exec on OpenShift
 
-In the output you will get link to the deployed Eclipse Che project. Use it to login to Eclipse Che.
-> Notice: for ocp.sh you could use argument `--setup-ocp-oauth`, but in this case you should use "Openshift v3" auth on the login page.
+1. [Build Eclipse Che Assembly](#building-eclipse-che-assembly).
 
-Register new user on the login page. After login you will be redirected to
-the Eclipse Che user dashboard.
+2. Deploy Eclipse Che on OpenShift ([example templates](https://github.com/eclipse/che/blob/master/deploy/openshift/)).
+   The output contains a link to the deployed Eclipse Che project. Use it to log in to Eclipse Che.
+> Note: For `ocp.sh`, the `--setup-ocp-oauth` is available. However, in this case, use "Openshift v3" authentication on the login page.
 
-Create an Eclipse Che 7.x workspace using the default Theia IDE. Then you can [test che-machine-exec with help che-theia-terminal-extension](#test-che-machine-exec-with-help-eclipse-che-theia-terminal) and [test che-machine-exec with help che-theia-task-plugin](#test-che-machine-exec-with-help-che-theia-task-plugin)
+3. Register a new user on the login page. After login, you are redirected to the Eclipse Che user dashboard.
 
-## Test on Minishift
+4. Create an Eclipse Che 7.x workspace using the default Che-Theia IDE. Then [test che-machine-exec using che-theia-terminal-extension](#testing-che-machine-exec-using-che-theia-terminal) and [test che-machine-exec using che-theia-task-plugin](#testing-che-machine-exec-using-che-theia-task-plugin).
 
-First, [build Eclipse Che Assembly](#build-eclipse-che-assembly).
+## Testing che-machine-exec on OpenShift using Minishift
 
-Install minishift with help this instractions:
- - https://docs.okd.io/latest/minishift/getting-started/preparing-to-install.html
- - https://docs.okd.io/latest/minishift/getting-started/setting-up-virtualization-environment.html
+1. [Build Eclipse Che Assembly](#building-eclipse-che-assembly).
 
-Install oc tool: [download oc binary for your platform](https://github.com/openshift/origin/releases), extract and apply this binary path to the system environment variables PATH. After that oc become availiable from terminal:
+2. Install Minishift using the following instractions:
+   - [Preparing to Install Minishift](https://docs.okd.io/latest/minishift/getting-started/preparing-to-install.html)
+   - [Setting Up the Virtualization Environment](https://docs.okd.io/latest/minishift/getting-started/setting-up-virtualization-environment.html)
 
-```bash
-$ oc version
-oc v3.9.0+191fece
-kubernetes v1.9.1+a0ce1bc657
-features: Basic-Auth GSSAPI Kerberos SPNEGO
-```
+3. Install the `oc` tool:
+  1. [Download the `oc` binary for your platform](https://github.com/openshift/origin/releases).
+  2. Extract and apply this binary path to the `PATH` system environment variable.
+  3. The `oc` tool is now availiable from the terminal:
+ ```
+ $ oc version
+ oc v3.9.0+191fece
```

Comment:
Kinda tells you how old this is when it refers to 3.9.0.

https://mirror.openshift.com/pub/openshift-v3/clients/3.11.213/ is the latest 3.x :D